### PR TITLE
peers: move to maxAge instead of timeslotting

### PR DESCRIPTION
**Pull request by robertkowalski**
rebased and incorporated feedback from #45

todo:
 - [x] switch git master of dht in package.json to release
 - [x] final testing

### CODE REVIEW

**Comment by mafintosh** [comment]
@robertkowalski dht out with the maxAge option, we should bump the semver to ^7.9.0

File: README.md [hunk 1]
```diff
@@ -29,18 +29,18 @@ Usage: grape --dp <dht-port> --aph <http-api-port> --bn <nodes> [--b
...
-  --help                    Show help                                  [boolean]
-  --version                 Show version number                        [boolean]
+  -b, --bind                 Listening host                             [string]
+  --dp, --dht_port           DHT listening port              [number] [required]
+  --dc, --dht_concurrency    DHT concurrency                            [number]
+  --dht_maxTables            DHT max tables                             [number]
+  --dht_maxValues            DHT max values                             [number]
+  --bn, --bootstrap          Bootstrap nodes                 [string] [required]
+  --aph, --api_port          HTTP api port                   [number] [required]
+  --dht_peer_maxAge, --dts   Max age for peers in DHT                   [number]
```
Comment (prdn) [inline README.md]: @robertkowalski we should probably use `--dpma` or `dpa` as alias

**Comment by robertkowalski** [comment]
@mafintosh bumped! @prdn changed!

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   ],
   "dependencies": {
     "async": "^2.6.0",
-    "bittorrent-dht": "^7.8.2",
+    "bittorrent-dht": "7.9.0",
```
Comment (mafintosh) [inline package.json]: we should still have the `^` here

File: lib/Grape.js [hunk 3]
```diff
@@ -219,6 +204,7 @@ class Grape extends Events {
     const kcnt = this.cbq0.cnt(ih)
     if (kcnt > 1) return
 
+    this._mem.remove(ih)
```
Comment (robertkowalski) [inline lib/Grape.js]: @mafintosh hmm

File: lib/Grape.js [hunk 4]
```diff
@@ -25,11 +25,12 @@ class Grape extends Events {
       dht_concurrency: 24,
       dht_nodeLiveness: 5 * 60 * 1000, // 5 min
       api_port: null,
-      timeslot: 10000
+      dht_peer_maxAge: 10000
```
Comment (prdn) [inline lib/Grape.js]: @robertkowalski I think this should be 60000 as default. 10000 is a bit too low.

**Pull request merged**